R/reportEmi.R

```diff
@@ -2204,9 +2528,12 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
 
 
 
-  ## 8. Emissions w/o non-energy use ----
-  # (Note: The non-energy use variables are so far only available for REMIND-EU runs and industry fixed_shares)
+  # 8. Ad-hoc fix for emissions w/o non-energy use and Aggregation to global and regional values  ----
+
+if (is.null(vm_demFENonEnergySector) && (module2realisation["industry", 2] == "fixed_shares")) {
```

Comment:
This condition breaks reportEmi for some project-specific reportings (for example Ariadne) and will be reverted.